services/speech_to_text/v1-corpus-builder.js

```diff
@@ -18,9 +18,11 @@ module.exports = function (RED) {
   const SERVICE_IDENTIFIER = 'speech-to-text';
   var temp = require('temp'),
     fs = require('fs'),
+    fsp = require('fs').promises,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 4.](https://app.codacy.com/manual/BetaWorks-NodeRED-Watson/node-red-node-watson/pullRequest?prid=4804371)